app/src/pages/outside/loginPage/loginPage.jsx

```diff
@@ -39,9 +42,10 @@ const cx = classNames.bind(styles);
   errorAuth,
   multipleAuth,
 }))
-@connect(null, {
+@connect((state) => ({
   showDefaultErrorNotification,
-})
+  instanceType: instanceTypeSelector(state),
+}))
```

Comment:
You've combined `mapStateToProps` and `mapDisoatchToProps` in one function, please split it to make it work as expected.